example/55-managedseed-gardenlet.yaml

```diff
@@ -0,0 +1,113 @@
+          backup:
+            provider: gcp
+    #       providerConfig:
+    #         <some-provider-specific-config-for-the-backup-buckets>
+    #       region: europe-north1
+            secretRef: {} # If empty, <namespace>/backup-<name> will be used as default
+    #         name: backup-my-managed-seed
+    #         namespace: garden
+          dns:
+            ingressDomain: ingress.dev.my-seed.example.com # Must be a subdomain of the shoot DNS domain, will be initialized automatically if empty and `.spec.ingress` is not set
+    #       provider: Must match the shoot DNS settings, will be initialized automatically if empty and `.spec.ingress` is set
+    #         type: aws-route53
+    #         secretRef:
+    #           name: ingress-secret
+    #           namespace: garden
+    #         domains:
+    #           include:
+    #           - my-custom-domain.com
+    #           - my-other-custom-domain.com
+    #           exclude:
+    #           - yet-another-custom-domain.com
+    #         zones:
+    #           include:
+    #           - zone-id-1
+    #           exclude:
+    #           - zone-id-2
+          networks:
+            pods: 10.241.128.0/17 # Must be the same as the shoot pods CIDR, will be initialized automatically if empty
+            services: 10.241.0.0/17 # Must be the same as the shoot services CIDR, will be initialized automatically if empty
+    #       nodes: 10.240.0.0/16 # Must be the same as the shoot nodes CIDR, will be initialized automatically if empty
+    #       shootDefaults:
+    #         pods: 100.96.0.0/11
+    #         services: 100.64.0.0/13
+    #       blockCIDRs:
+    #       - 169.254.169.254/32
+          provider:
+            type: gcp # Must be the same as the shoot provider type, will be initialized automatically if empty
+            region: europe-west1 # Must be the same as the shoot region, will be initialized automatically if empty
+    #     secretRef:
+    #       name: seed-my-managed-seed
+    #       namespace: garden
+    #     settings:
+    #       excessCapacityReservation:
+    #         enabled: true
+    #       scheduling:
+    #         visible: true
+    #       shootDNS:
+    #         enabled: true
+    #       loadBalancerServices:
+    #         annotations:
+    #           foo: bar
+    #       verticalPodAutoscaler:
+    #          enabled: true
+    #     taints:
+    #     - key: seed.gardener.cloud/protected # protected
+    #     - key: <some-key>
+    #     volume:
+    #       minimumSize: 20Gi
+    #     ingress: # If set then `.spec.dns.ingressDomain` must not be set and `.spec.dns.provider` must be set.
+    #       domain: ingress.dev.my-seed.example.com # Must be a subdomain of the shoot DNS domain, will be initialized automatically if empty
+    #       controller:
+    #         kind: nginx
+    #         providerConfig:
+    #           <some-optional-config-for-the-nginx-ingress-controller>
```

Comment:
To prevent double maintenance, instead of duplicating all of this here, shall be simply "refer" to the `GardenletConfiguration` example?

Reply:
OK, I still have my doubts but I did as requested.